cores/esp8266/core_esp8266_main.cpp

```diff
@@ -480,6 +480,11 @@ extern "C" void user_init(void) {
 
     uart_div_modify(0, UART_CLK_FREQ / (115200));
 
+#if FLASH_MAP_SUPPORT
+    const char *err_msg = flashinit();
+    if (err_msg) __unhandled_exception(err_msg);
```

Comment:
The message text looks like this when printed. Is there any objection to the "Unhandled C++ exception" choice in the postmortem? I didn't see any others that would work well for a single string.
```
Unhandled C++ exception: autoconfig.ino.ino:15 flashinit: configuration not found
```

Reply:
could `__panic__func()` or `__assert_func()` help ?

Reply:
If I change postmortem to print panics when `s_panic_line` is set to 0. That seems to work. I'll push the change.